includes/class-wc-stripe-helper.php

```diff
@@ -214,6 +215,10 @@ public static function get_localized_messages() {
 				'invalid_request_error'    => is_add_payment_method_page()
 					? __( 'Unable to save this payment method, please try again or use alternative method.', 'woocommerce-gateway-stripe' )
 					: __( 'Unable to process this payment, please try again or use alternative method.', 'woocommerce-gateway-stripe' ),
+				'amount_too_large'         => __( 'The order total is too high for this payment method', 'woocommerce-gateway-stripe' ),
+				'amount_too_small'         => __( 'The order total is too low for this payment method', 'woocommerce-gateway-stripe' ),
+				'country_code_invalid'     => __( 'Invalid country code, please try again with a valid country code', 'woocommerce-gateway-stripe' ),
```

Comment:
@leonardola couple of questions here:

1. With UPE, when selecting a country and state that's not from Brazil, this message shows up. I think it's not immediately clear what the problem is for a shopper, but I don't see a specific code returned by Stripe to handle this like other error codes. Do you think we could improve it?
![image](https://user-images.githubusercontent.com/41606954/142065101-c5c38bc7-bcfe-4e36-a51c-51e34305f8f3.png)
 
2. Are we actually using this string in the checkout when that error is returned by stripe? With UPE enabled, when I select a billing country that's not Brazil (and add "AC" in the state to skip the validation I mentioned before), I see the message returned by the request instead of "Invalid country code, please try again with a valid country code". Even though the message returned by Stripe could be useful, it'd be good to have a translatable string.
![image](https://user-images.githubusercontent.com/41606954/142059206-b8e64c38-fbc8-4ace-b20b-577ee234821a.png)